recipe/meta.yaml

```diff
@@ -24,13 +24,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
   host:
     - libqdldl ={{ qdldlversion }}
+    - numpy # For unittests
+    - scipy =1.11.4 # For unittests, as of 0.6.3 imports randn from scipy
 
 test:
+  source_files:
+    - build/out/osqp_demo*
+    - build/out/osqp_tester*
   commands:
+    - build\out\osqp_demo.exe  # [win]
+    - build\out\osqp_tester.exe  # [win]
+    - ./build/out/osqp_demo  # [not win]
+    - ./build/out/osqp_tester  # [not win]
```

Comment:
Please move this to the end of the test section. Tests should go in increasing order of complexity - first simple file existence checks (if those fail, more complicated ones make no sense), then more integrated workflows.

Reply:
That makes sense, thank you.